src/auth/passwordreset/redux/validations.js

```diff
@@ -0,0 +1,4 @@
+const loginUtil = {
+  LOGIN_UTILS: /^[a-zA-Z0-9]+@+[a-zA-Z0-9]+.+[A-z]/,
```

Comment:
consider restructuring this regex as it happens to flag some correct emails like the ones in a format of **name.fname@host.com**. try using this regex `/^[^\s@]+@[^\s@]+\.[^\s@]+$/`

Reply:
This has been resolved, thank you.